deps.bzl

```diff
@@ -760,8 +760,9 @@ def prysm_deps():
         patches = [
             "//third_party:com_github_ethereum_go_ethereum_secp256k1.patch",
         ],
-        sum = "h1:8hl57x77HSUo+cXExrURjU/w1VhL+ShCTJrTwcCQSe4=",
-        version = "v1.14.12",
+        replace = "github.com/s1na/go-ethereum",
```

Comment:
This should be a proper release as we will use all the geth packages in production for prysm. Any reason we have to use a geth fork ?

Reply:
yeah we should wait but i couldn't test yet I think since it may not have been released, will try to revert to 1.14.13 and try that

Reply:
i just tested 1.14.13 and it fails minimal e2e electra is not enabled there only on sina's branch